app/components/ui/copy-on-launch-modal/component.js

```diff
@@ -32,11 +29,9 @@ export default Component.extend({
           if (originalTale) {
               
               let handleLaunchError = (tale, err) => {
-                // deal with the failure here
-                //tale.set('launchStatus', 'error');
-                //tale.set('launchError', err.message || err);
-                
                 console.error('Failed to launch Tale', err);
+                self.set("errorMessage", err.message);
+                $('.ui.modal.compose-error').modal('show');
```

Comment:
Possibly a copy-paste error here. I think it should be:
```suggestion
                $('.ui.modal.copy-error').modal('show');
```

Reply:
Oops thanks for catching that, apologies for having to deal with the mismatch.